mteb/tasks/Retrieval/multilingual/MIRACLRetrieval.py

```diff
@@ -337,6 +340,71 @@ class MIRACLRetrievalHardNegatives(MultilingualTask, AbsTaskRetrieval):
   volume = {11},
   year = {2023},
 }
+""",
+        adapted_from=["MIRACLRetrieval"],
+    )
+
+    def load_data(self, **kwargs):
+        if self.data_loaded:
+            return
+
+        self.corpus, self.queries, self.relevant_docs = (
+            _load_miracl_data_hard_negatives(
+                path=self.metadata.dataset["path"],
+                langs=self.hf_subsets,
+                splits=self.metadata_dict["eval_splits"],
+                cache_dir=kwargs.get("cache_dir", None),
+                revision=self.metadata.dataset["revision"],
+                trust_remote_code=self.metadata.dataset.get("trust_remote_code", False),
+            )
+        )
+
+        self.data_loaded = True
+
+
+class MIRACLRetrievalHardNegativesV2(MultilingualTask, AbsTaskRetrieval):
+    metadata = TaskMetadata(
+        name="MIRACLRetrievalHardNegatives.v2",
+        description=(
+            "MIRACL (Multilingual Information Retrieval Across a Continuum of Languages) is a multilingual retrieval "
+            "dataset that focuses on search across 18 different languages. The hard negative version has been "
+            "created by pooling the 250 top documents per query from BM25, e5-multilingual-large and e5-mistral-instruct."
+            "V2 version differs only by prompt."
+        ),
+        reference="http://miracl.ai",
```

Comment:
should we define this in a dictionary outside and unpack? (to avoid changes in one that doesn't happen in the other)